src/module-analyzer.js

```diff
@@ -274,6 +275,9 @@ export class ModuleAnalyzer {
         } else if (conventional = BindingBehaviorResource.convention(key)) {
           resources.push(new ResourceDescription(key, exportedValue, conventional));
           metadata.define(metadata.resource, conventional, exportedValue);
+        } else if (conventional = ViewEngineHooksResource.convention(key)) {
```

Comment:
@davismj the following 2 lines are exactly as the above, so why not move this to the previous `else-if`?